hedera-node/docker/start-services.sh

```diff
@@ -68,4 +68,7 @@ if [[ "${CI_WAIT_FOR_PEERS}" = true && -n "${NODE_ID}" ]]; then
 fi
 
 java -cp 'data/lib/*' -Dflag=1 -Dfile.encoding='utf-8' \
+  --add-opens java.base/jdk.internal.misc=ALL-UNNAMED \
+  --add-opens java.base/java.nio=ALL-UNNAMED \
+  -Dio.netty.tryReflectionSetAccessible=true \
```

Comment:
Do we need to ensure the production containers use these options to bypass the netty JPMS issues? If so, then a ticket should be sent to DevOps for the appropriate NMT changes to make these the default.

Reply:
We will.  Where do I enter that ticket?